department-of-veterans-affairs/vets-website
# Update uiSchema titles to titleUI pattern

## Summary
Update uiSchema titles on the Pension Benefits form (21P-527EZ)

### Description
Update all uiSchema title's to use the `titleUI` web component pattern to improve consistency and maintainability. The `titleUI` web component pattern accepts a title and a description that is inserted into the form's fieldset legend as an h3 header and optional description that is announced by screen readers when navigating through the application.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Complete the application and verify the styling of the form header
3. Verify that the screen reader announces the title and description of the form page when the page is loaded
4. Verify that the application state and data is saved to redux and submitted to the api
5. Verify that the application is able to be submitted

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74383)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_financial_total-net-worth](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/479b7480-bad4-47c4-a8f4-ea1efce7ca21)|![localhost_3001_pension_application_527EZ_form-saved (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/853273a9-d8fe-4d9b-8d08-7063caf01aba)


## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
